frontend/src/App.tsx

```diff
@@ -82,7 +82,7 @@ const App = (): React.ReactElement => {
                   component={TeacherPage}
                   exact
                   path={Routes.TEACHER_LANDING}
-                  roles={["Teacher"]}
+                  roles={["Teacher", "Admin"]}
```

Comment:
does it make sense to allow admins to see the teacher page?

Reply:
no, i think we just left that in there for now so you dont have to logout to see other pages

Reply:
but also shouldn't admins have access the everything?